model/ftn/w3pro3md.ftn

```diff
@@ -960,7 +960,7 @@
       IF ( FLCUR ) THEN
 !/T          WRITE (NDST,9022)
 !
-!/OMPH/!$OMP PARALLEL DO PRIVATE (IXY)
+!/OMPH/!$OMP PARALLEL DO PRIVATE (ISEA,IXY)
```

Comment:
I believe that loop variables (ISEA in this case) are automatically made private by the OMP implementation.
Not that it is necessarily a bad thing to be explicit...

Reply:
That's correct, our assumption was that it would be best to make this clear.